Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/Pages/TaskList/SchoolAndTrustInformation/Form7ReceivedIntegrationTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using AutoFixture;
+using Dfe.PrepareConversions.Tests.Extensions;
+using FluentAssertions;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfe.PrepareConversions.Tests.Pages.TaskList.SchoolAndTrustInformation
+{
+   public class Form7ReceivedIntegrationTests : BaseIntegrationTests
+   {
+      public Form7ReceivedIntegrationTests(IntegrationTestingWebApplicationFactory factory) : base(factory) { }
+
+      [Fact]
+      public async Task Should_navigate_to_and_update_form_7_received()
+      {
+         var project = AddGetProject(p => p.ApplicationReceivedDate = null);
+         AddPatchConfiguredProject(project, x =>
+         {
+            x.Form7Received = "Yes";
+            x.Urn = project.Urn;
+         });
+
+         await OpenAndConfirmPathAsync($"/task-list/{project.Id}/confirm-school-trust-information-project-dates");
+         await NavigateAsync("Change", 2);
+
+         Document.Url.Should().BeUrl($"/task-list/{project.Id}/confirm-school-trust-information-project-dates/form-7-received");
+         // Do the input boxes on the page default to none selected when coming from an empty value
+         // Yes 
+         Document.QuerySelector<IHtmlInputElement>("#form-7-received").IsChecked.Should().BeFalse();
+         // No
+         Document.QuerySelector<IHtmlInputElement>("#form-7-received-2").IsChecked.Should().BeFalse();
```

Comment:
Use the value as a suffix, instead of numerics will make this make more sense (and remove the need for comments to explain)

Reply:
Created tech debt https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_backlogs/backlog/Prepare%20conversions%20and%20transfers/Features/?workitem=122212